sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/OpenCoverReportParserTest.java

```diff
@@ -185,6 +185,14 @@ public void branchCoverage_getter_setter_complex_case() throws Exception {
           // line 21: first line inside BodyMethod - 3 statements (what is after 'goto' is ignored)
           new BranchCoverage(21, 3, 3)
         );
+
+      List<String> traceLogs = logTester.logs(LoggerLevel.TRACE);
+      assertThat(traceLogs).hasSize(35);
+
+      String logLine = traceLogs.get(32);
+      assertThat(logLine)
+        .startsWith("Found coverage information about '10' lines having multiple sequence points for file '")
```

Comment:
actually... it's about sequence points on a single line, that do not span on multiple lines